app/models/manageiq/providers/redfish/physical_infra_manager/physical_server.rb

```diff
@@ -3,5 +3,10 @@ class PhysicalInfraManager::PhysicalServer < ::PhysicalServer
     def self.display_name(number = 1)
       n_('Physical Server (Redfish)', 'Physical Servers (Redfish)', number)
     end
+
+    def provider_object(connection = nil)
```

Comment:
Do we need this `nil` here? Since we require it to be non-`nil`, we might as well require `connection` parameter.

Reply:
Good point - and as soon as we then made `connection` a required parameter, we were safe to drop the `raise` statement, because it's kinda obvious that required parameter is not supposed to be nil. I've pushed the update.